Sources/ScipioKit/Producer/PIF/ToolchainGenerator.swift

```diff
@@ -87,3 +87,36 @@ struct ToolchainGenerator {
         #endif
     }
 }
+
+fileprivate extension ToolchainGenerator {
+
+    /// A non-caching environment-aware implementation of `SwiftSDK.sdkPlatformFrameworkPaths`
+    /// Returns `macosx` sdk platform framework path.
+    func nonCachingSDKPlatformFrameworkPaths() async throws -> (fwk: AbsolutePath, lib: AbsolutePath) {
```

Comment:
As I mentioned in the PR description, this is a copy of [the existing function](https://github.com/swiftlang/swift-package-manager/blob/release/6.0/Sources/PackageModel/SwiftSDKs/SwiftSDK.swift#L592-L595) without the path-caching part.

Reply:
Could you note this context in the comment?

Reply:
Please avoid abbreviations `frameworkPath`, `libPath`

Reply:
I prefer to add a verb to the name. This method executes external commands, so it's not better to name it like a getter.

 How about renaming like `resolveSDKPlatformFrameworkPaths`?

Reply:
> Please avoid abbreviations frameworkPath, libPath

Sorry, I just copied Apple's implementation one-to-one. Of course, it makes sense. Done.

Reply:
> How about renaming like resolveSDKPlatformFrameworkPaths?

Yes, I also like it. Done.

Reply:
> Could you note this context in the comment?

> /// A non-caching environment-aware implementation of `SwiftSDK.sdkPlatformFrameworkPaths`

I believe it's already in the comment above, or did you mean something else?